skrub/_datetime_encoder.py

```diff
@@ -69,7 +70,7 @@ def _get_dt_feature_polars(col, feature):
     return getattr(col.dt, feature)()
 
 
-class DatetimeEncoder(SingleColumnTransformer):
+class DatetimeEncoder(TransformerMixin, SingleColumnTransformer):
```

Comment:
@jeromedockes I just noticed the `SingleColumnTransformer` doesn't inherit from `TransformerMixin`, is there a reason for that?

Reply:
`OnEachColumn` is the one implementing `fit_transform` and thus inherit from `TransformerMixin`. `SingleColumnTransformer` does not implement `transform` so per se, it makes sense to not inherit from `TransformerMixin`.

Reply:
Mainly for the tags infrastructure. Otherwise, we need to set manually the tags related to `transformer_tags` ourselves.